spec/system/admin/orders_spec.rb

```diff
@@ -554,6 +554,8 @@
           expect(page).to have_content "This will cancel the current order."
 
           within ".reveal-modal" do
+            # Untick SEND A CANCELLATION EMAIL TO THE CUSTOMER
+            page.find("input[id='send_cancellation_email']").click
```

Comment:
I think that we can use a common Capybara helper for this:
```suggestion
            uncheck "Send a cancellation email to the customer"
```

Reply:
TIL about `uncheck` , thanks it looks much better !